mlrun/k8s_utils.py

```diff
@@ -269,6 +271,49 @@ def get_logger_pods(self, uid, namespace=''):
 
         return results
 
+    def get_shell_pod_name(self, pod_name='shell'):
```

Comment:
there is no room for `shell` specific function in the k8s class, i also assume the shell has labels and we can find it based on the label vs a loop